homeassistant/components/generic_hygrostat/humidifier.py

```diff
@@ -357,7 +357,6 @@ async def _async_operate(self, time=None, force=False):
                 self._target_humidity,
             ):
                 self._active = True
-                force = True
```

Comment:
This is not the right place to do it. This condition is true when the generyc_hygrostat obtains the initial humidity values.
Instead, remove `force=True` from `await self._async_operate(force=True)` on line 285 in `async_set_humidity()`.
Also, the `test_init_ignores_tolerance` test should not be removed.